core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -276,6 +290,9 @@ const getSkillEditorUrl = async function(browser, page) {
   } catch (e) {
     // eslint-disable-next-line no-console
     console.log(e);
+    // eslint-disable-next-line no-console
+    console.log(e.stack);
```

Comment:
Do we need to log both of these? Shouldn't one be enough?

Reply:
Ah good catch, thanks -- I did this when debugging and forgot to remove it. Done.